src/sass/accordion/accordion.scss

```diff
@@ -0,0 +1,63 @@
+@import "../variables/variables";
+@import "../mixins/private/token-mixins";
+
+ul.accordion {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+}
+
+ul.accordion li:not(:last-child) {
+    border-bottom: 1px solid var(--color-stroke-subtle);
+}
+
+ul.accordion summary.details__summary {
+    border-radius: 0;
+    display: flex;
+    font-size: var(--font-size-medium);
+    justify-content: space-between;
+    min-height: 48px;
+    padding: 12px 16px;
+}
+
+ul.accordion details .details__content {
+    margin: 0 16px 6px;
+    opacity: 0;
+    transform: scaleY(0);
+    transform-origin: top;
+    transition:
+        opacity 500ms cubic-bezier(0.3, 0, 0, 1),
+        transform 500ms cubic-bezier(0.3, 0, 0, 1);
```

Comment:
Did we get this from DS? It doesn't feel very natural. I tried using this and it works more fluidly:

```css
opacity .5s cubic-bezier(.3, 0, 0, 1),
transform .150s cubic-bezier(.3, 0, 0, 1)
```

Reply:
All design suggested is as shown in below image,
<img width="239" alt="image" src="https://github.com/user-attachments/assets/0925cb01-8704-48d3-b145-5a23ef601ea9">

Reply:
I would say we can push current changes to skin in this release, and address this in next release with ebay UI. We can ask DS team to review and update the animations.

Reply:
Ok, let's leave the timing function in place for now.